crates/node/src/service.rs

```diff
@@ -527,6 +531,15 @@ pub fn new_full(
         );
     }
 
+    if let Some(ref l1_messages_config) = l1_messages {
```

Comment:
Should it be spawned before or after aura?

Reply:
I do not see any valid point for spawning this service before `aura` that is responsible for block production. In my opinion it should start after `aura` while this thread submits transactions to be proceeded.